spring-kafka/src/test/java/org/springframework/kafka/core/KafkaStreamsCustomizerTests.java

```diff
@@ -33,6 +33,7 @@
 import org.springframework.kafka.annotation.EnableKafkaStreams;
 import org.springframework.kafka.annotation.KafkaStreamsDefaultConfiguration;
 import org.springframework.kafka.config.KafkaStreamsConfiguration;
+import org.springframework.kafka.config.StreamsBuilderFactoryBean;
```

Comment:
It's not so critical, but may be would be better to move relevant tests to the `config` package, too.